docs/maintainer-build-platforms

```diff
@@ -0,0 +1,49 @@
+# Build Platforms
+
+The LinuxKit team maintains several build platforms. These are donated by Equinix Metal (arm64) and IBM (s390x).
+
+## arm64
+
+The arm64 platform is fully supported by most OS vendors and Docker. Just upgrade to the latest OS and install the latest Docker using the
+packaging tools. As of this writing, that is Ubuntu with `apt`.
```

Comment:
I think Docker recommends using the docker install from their channels rather than using the distro packages. I think it's better since you have better control over which version of docker you are using. https://docs.docker.com/engine/install/

Reply:
Yeah, I was implying that, but wasn't too clear. I really just wanted to capture the messy s390x steps before I forgot. I will update it.